validation/src/main/java/org/ehrbase/validation/constraints/wrappers/CDuration.java

```diff
@@ -37,15 +37,12 @@
  */
 public class CDuration extends CConstraint implements I_CTypeValidate {
 
-    CDuration(Map<String, Map<String, String>> localTerminologyLookup) {
+    protected CDuration(Map<String, Map<String, String>> localTerminologyLookup) {
         super(localTerminologyLookup);
     }
 
     @Override
-    public void validate(String path, Object aValue, CPRIMITIVE cprimitive) throws IllegalArgumentException {
-        if (!(aValue instanceof OffsetDateTime))
```

Comment:
@chevalleyc isn't the type check needed here?

Reply:
This is now done by Archie (if the string cannot be parsed, it is null and rejected by the validator)